README.md

```diff
@@ -217,61 +230,292 @@ Thus, this allows you to have many vendored tarballs by using the `--tag` parame
 > it will produce a vendored tarball. Otherwise, although it is *somewhat* correct to
 > say `vendor-utils.tar.zst` should be the produced tarball, in reality, it won't exist
 > because it has [NO dependencies](https://github.com/ibm-s390-linux/s390-tools/blob/master/rust/utils/Cargo.toml) after all.
- 
-# Parameters
 
+# Cargo Vendor Home Registry
+
+> [!NOTE]
+> This attempt started in this repository <https://github.com/openSUSE-Rust/obs-service-cargo-vendor-home-registry> but now,
+> it's been merged here to avoid maintenance burden. As the old repository will retire, it still remains there for those that
+> are curious about how we go from there to here.
+
+As previously mentioned, we talked about the usage of tags for multi vendor scenarios. However, to avoid this issue of having
+to write a long service file while also managing multiple vendors, we introduced the vendoring of `$CARGO_HOME`,
+specifically, `$CARGO_HOME/registry`. The service file will look like this
+
+```xml
+<services>
+  <service name="download_files" mode="manual" />
+  <service name="cargo_vendor" mode="manual">
+     <param name="src">s390-tools*.tar.gz</param>
+	 <param name="update">true</param>
+	 <param name="method">registry</param>
+	 <param name="no_root_manifest">true</param>
+	 <param name="cargotoml">rust/pvsecret/Cargo.toml</param>
+	 <param name="cargotoml">rust/pv/Cargo.toml</param>
+	 <param name="cargotoml">rust/utils/Cargo.toml</param>
+  </service>
+  <service name="cargo_audit" mode="manual" />
```

Comment:
I think our examples no longer need the cargo_audit service written out (since cargo_vendor also audits now)

Reply:
Good to know. I guess that's old cruft now.

Reply:
Try ctrl+f to remove all of those

Reply:
I just used kakoune and selected it all :P